src/core/MOM_PressureForce_FV.F90

```diff
@@ -1940,9 +2058,14 @@ subroutine PressureForce_FV_init(Time, G, GV, US, param_file, diag, CS, SAL_CSp,
   call get_param(param_file, mdl, "RESET_INTXPA_INTEGRAL", CS%reset_intxpa_integral, &
                  "If true, reset INTXPA to match pressures at first nonvanished cell. "//&
                  "Includes pressure correction.", default=.false., do_not_log=.not.use_EOS)
+  call get_param(param_file, mdl, "RESET_INTXPA_INTEGRAL_FLATTEST", CS%reset_intxpa_flattest, &
+                 "If true, use flattest layer as reference layer where there is no "//&
```

Comment:
We are searching for the flattest interface, rather than a layer.  I don't know what a flat layer is, but the meaning of the flattest interface is clear enough.  Please modify this parameter description accordingly (or we could modify this for you).